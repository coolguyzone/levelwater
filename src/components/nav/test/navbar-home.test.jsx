import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NavbarHome from '../navbar-home.jsx';

describe('navbar home', () => {
  it('renders a snapshot', () => {
    const tree = shallow(<NavbarHome/>);
    expect(toJson(tree)).toMatchSnapshot();
  });
});
