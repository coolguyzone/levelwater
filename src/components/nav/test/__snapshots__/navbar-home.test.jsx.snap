exports[`navbar home renders a snapshot 1`] = `
<div
  id="navbar">
  <div
    className="ui grid">
    <div
      className="three wide column"
      id="demo-button">
      <Connect(DemoButton) />
    </div>
    <div
      className="nine wide column"
      id="header">
      <Link
        replace={false}
        to="/">
        <h1>
          levelwater.io
        </h1>
      </Link>
    </div>
    <div
      className="four wide column"
      id="right-nav-home">
      <RightNavHome />
    </div>
  </div>
  <div>
    <MobileButtons />
  </div>
</div>
`;
